client/data/promote-post/use-promote-post-campaigns-query.ts

```diff
@@ -43,6 +43,21 @@ export type CampaignResponse = {
 		conversion_rate?: number;
 		conversion_value?: Record< string, number >;
 		conversion_last_currency_found?: string;
+		tsp?: {
+			impressions_total: number;
+			clicks_total: number;
+			permalink: string;
+			likes_total: number;
+			replies_total: number;
+			replies?: {
+				total_notes: number;
+				notes: {
+					blog_name: string;
+					type: string;
+					blog_url: string;
```

Comment:
Could you add `reply_text?: string;` to place actual comment?

Reply:
added